recipes/gmpyinfr_dbutils/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gmpyinfr_dbutils" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gmpyinfr_dbutils-{{ version }}.tar.gz
+  sha256: 1e4c000262e157d48f345ab0a3a734269c69af4842d80d6b8ff48dd55e071fb7
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or osx or py<3.6 or py>3.8]
```

Comment:
This looke like a `noarch: python` package. What is the reason to limit the Python version so hard?